apps/common-app/src/examples/RuntimeTests/ReJest/RuntimeTestsRunner.tsx

```diff
@@ -195,26 +198,43 @@ function SelectAllButtonProps({ handleSelectAllClick, select }: SelectAllButtonP
       onPressIn={handleSelectAllClickIn}
       onPressOut={() => handleSelectAllClickOut()}
       style={[styles.selectAllButton, isPressed ? styles.pressedButton : {}]}>
-      <Text style={styles.buttonText}>{select ? 'Select all' : 'Deselect all'}</Text>
+      <Text style={styles.navyText}>{select ? 'Select all' : 'Deselect all'}</Text>
     </Pressable>
   );
 }
 
+const TEXT = {
+  fontSize: 20,
+  alignSelf: 'center',
+} as const;
+
+const BUTTON = {
+  height: 40,
+  justifyContent: 'center',
+  alignItems: 'center',
+} as const;
+
+const WHITE_BUTTON = {
+  ...BUTTON,
+  borderWidth: 2,
+  backgroundColor: 'white',
+  borderColor: 'navy',
+  marginVertical: 5,
+  borderRadius: 10,
+} as const;
+
```

Comment:
You can pass an array of styles instead, i.e.
`style={[styles.button, styles.buttonWhite])` or something like this. This doesn't look nice :/